lib/datadog/core/configuration/settings.rb

```diff
@@ -981,6 +981,33 @@ def initialize(*_)
           end
         end
 
+        # Datadog Error Tracking-specific configurations
+        #
+        # @public_api
+        settings :error_tracking do
+          # Enable automatic reporting of handled errors and set the scope
+          # of the errors to report: all | user | third_party
+          #
+          # @default 'DD_ERROR_TRACKING_HANDLED_ERRORS' environement variable, otherwise `nil`
+          # @return [String, nil]
+          option :instrumentation_scope do |o|
```

Comment:
Can you add a validation here for the 3 valid values, similar to this OpenSearch option:
https://github.com/DataDog/dd-trace-rb/blob/19100f658ce49b14a0413361ff64f62280411058/lib/datadog/tracing/contrib/opensearch/configuration/settings.rb#L54-L64

Reply:
You can then likely remove this check:
```ruby
            return if !settings.error_tracking.instrumentation_scope.empty? &&
              !['all', 'user', 'third_party'].include?(settings.error_tracking.instrumentation_scope)
```

Reply:
addressed in ecfdce57e4880c87d13c07923e31ef91a92f64e7